integration_tests/test_dbapi.py

```diff
@@ -106,22 +106,6 @@ def test_string_query_param(trino_connection):
     assert rows[0][0] == "six'"
 
 
-def test_datetime_query_param(trino_connection):
```

Comment:
I've added extended test case scenario at the bottom - with and without precision